lib/determine-basal/determine-basal.js

```diff
@@ -374,10 +374,20 @@ var determine_basal = function determine_basal(glucose_status, currenttemp, iob_
             }
         }
     }
+  
+    var minutes_running;
+    if (typeof currenttemp.duration == 'undefined' || currenttemp.duration == 0) {
+        minutes_running = 0;
```

Comment:
I believe in this case it does not matter as a duration of 0 or unedfined means that no temp is running at all.
But yes, better use the logical default value in case there are some corner cases or it it might get used in other places in the future.